include/boost/geometry/algorithms/detail/distance/segment_to_box.hpp

```diff
@@ -380,24 +379,37 @@ class segment_to_box_2D
                     // segment & crosses band (TODO:merge with box-box dist)
                     if (math::equals(geometry::get<0>(p0), geometry::get<0>(p1)))
                     {
-                        SegmentPoint high = geometry::get<1>(p1) > geometry::get<1>(p0) ? p1 : p0;
-                        if (less_equal(geometry::get<1>(high), geometry::get<1>(top_right)))
+                        SegmentPoint high = geometry::get<1>(p1)
+                                            > geometry::get<1>(p0) ? p1 : p0;
+                        if (less_equal(geometry::get<1>(high),
+                                       geometry::get<1>(top_right)))
```

Comment:
I know that this is probably because of the max line lengths but IMO the original version was more readable. Are we very strict about line lengths?

Reply:
It was boarder-line w.r.t. max line length, so I revert the change since I also think is more readable this way.